nix/patches/cardano-sl.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/Pos/Util/Config.hs b/src/Pos/Util/Config.hs
```

Comment:
Naughty :-) but is an acceptable working solution.

Reply:
It's just a patch :-)